client/src/pages/ResetPasswordRequest.jsx

```diff
@@ -0,0 +1,123 @@
+import { Formik, Form, Field } from 'formik';
+import { Link } from 'react-router-dom';
+import cn from 'classnames';
+import { useState } from 'react';
+import { usePageError } from '../hooks/usePageError.js';
+import { authService } from '../services/authService.js';
+
+function validateEmail(value) {
+  if (!value) {
+    return 'Email is required';
+  }
+
+  const emailPattern = /^[\w.+-]+@([\w-]+\.){1,3}[\w-]{2,}$/;
+
+  if (!emailPattern.test(value)) {
+    return 'Email is not valid';
+  }
+}
+
+export const ResetPasswordRequest = () => {
+  const [emailSent, setEmailSent] = useState(false);
+  const [error, setError] = usePageError('');
+
+  if (emailSent) {
+    return (
+      <section className="">
+        <h1 className="title">Check your email</h1>
+        <p>We have sent you following instructions</p>
```

Comment:
There is a typo in the text 'We have sent you following instructions'. It should be 'We have sent you the following instructions'.